### PR TITLE
[Scorecards] Updated web copy for Combined Authority

### PR DESCRIPTION
**Pull request by lucascumsille**
This PR has some errors in the copy spotted by CEUK.
Part of: https://github.com/orgs/mysociety/projects/24/views/2?pane=issue&itemId=19970395

### CODE REVIEW

File: scoring/templates/scoring/methodology2023.html [hunk 1]
```diff
@@ -92,17 +92,17 @@ <h4 class="toggle-section__header">
...
+                                                            district council’s
                                                         {% elif council_type == 'county' %}
-                                                            county
+                                                            county council’s
                                                         {% elif council_type == 'northern-ireland' %}
-                                                            Northern Irish
+                                                            Northern Irish council’s
                                                         {% elif council_type == 'combined' %}
-                                                            Combined Authority
+                                                            Combined Authority's
```
Comment (zarino) [inline scoring/templates/scoring/methodology2023.html:103]: Would be nice if this were a curly quote `’` like the others.

File: scoring/views.py [hunk 2]
```diff
@@ -1380,7 +1380,7 @@ def get_context_data(self, **kwargs):
                             </ul>
                             """,
                         "clarifications": """
-                            <p>Marked mainly using  publicly available data on shared transport schemes.<a href="https://como.org.uk/" class="d-inline"> Coordinated Mobility's </a></p>
+                            <p>Marked mainly using  publicly available data on shared transport schemes from<a href="https://como.org.uk/" class="d-inline"> Coordinated Mobility</a></p>
```
Comment (zarino) [inline scoring/views.py:1383]: Bit odd that the space character is _inside_ the link element, rather than _before_ it.

**Review by zarino — APPROVED** [review]
Looks good. Just two very minor points.

**Comment by lucascumsille** [comment]
Thanks @zarino I just merged and deployed, including the changes above.

**Pull request merged**